pkg/cephmgr/osd/scheme.go

```diff
@@ -26,21 +26,29 @@ import (
 )
 
 const (
-	WalPartitionName      = "wal"
-	DatabasePartitionName = "db"
-	BlockPartitionName    = "block"
-	walPartition          = 1
-	databasePartition     = 2
-	blockPartition        = 3
-	UseRemainingSpace     = -1
-	schemeFilename        = "partition-scheme"
-	WalDefaultSizeMB      = 576
-	DBDefaultSizeMB       = 20480
+	Filestore                     = "filestore"
```

Comment:
Yeah. It is a bit different in Golang. You have to use const and iota. And then you can have a switch command to iterate through them. Here is an example in Kubernetes. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/pod_update.go#L40 and https://github.com/kubernetes/kubernetes/blob/master/pkg/util/iptables/iptables.go#L78